builtin/mainmenu/common.lua

```diff
@@ -128,44 +100,35 @@ function render_serverlist_row(spec, is_favorite)
 		elseif clients_percent <= 60  then clients_color = '#a1e587' -- 0-60%: green
 		elseif clients_percent <= 90  then clients_color = '#ffdc97' -- 60-90%: yellow
 		elseif clients_percent == 100 then clients_color = '#dd5b5b' -- full server: red (darker)
-		else				   clients_color = '#ffba97' -- 90-100%: orange
+		else							   clients_color = '#ffba97' -- 90-100%: orange
 		end
 
-		details = details .. clients_color .. ',' ..
-			render_client_count(spec.clients) .. ',/,' ..
-			render_client_count(spec.clients_max) .. ','
-
-	elseif grey_out then
-		details = details .. '#aaaaaa,?,/,?,'
+		table.insert(details, clients_color)
+		table.insert(details, spec.clients.." / "..spec.clients_max)
 	else
-		details = details .. ',?,/,?,'
+		table.insert(details, color)
+		table.insert(details, "?")
 	end
 
 	if spec.creative then
-		details = details .. "1,"
-	else
-		details = details .. "0,"
-	end
-
-	if spec.damage then
-		details = details .. "1,"
+		table.insert(details, "1")
+	elseif spec.pvp then
+		table.insert(details, "3")
+	elseif spec.damage then
+		table.insert(details, "2")
 	else
-		details = details .. "0,"
+		table.insert(details, "0")
 	end
 
-	if spec.pvp then
-		details = details .. "1,"
-	else
-		details = details .. "0,"
-	end
+	table.insert(details, color)
+	table.insert(details, text)
 
-	return details .. (grey_out and '#aaaaaa,' or ',') .. text
+	return table.concat(details, ",")
 end
 
---------------------------------------------------------------------------------
 os.tempfolder = function()
 	if core.settings:get("TMPFolder") then
-		return core.settings:get("TMPFolder") .. DIR_DELIM .. "MT_" .. math.random(0,10000)
+		return core.settings:get("TMPFolder") .. "/MT_" .. math.random(0,10000)
```

Comment:
Important for various OS.

Reply:
No, `DIR_DELIM` is being deprecated for good. Lua's `stdlib` *and* Minetest are both supposed to work with `"/"`, automatically converting it to the proper delimiter, as does LuaFileSystem. I see no issue here.

Reply:
There may still be bugs in Minetest that don't allow this. It's best for this to be fixed in another PR which sets DIR_DELIM to `/`